specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-02-01/managedClusters.json

```diff
@@ -0,0 +1,4120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2021-02-01"
```

Comment:
@bcho - For quicker API reviews, we recommend to fill the change log informing the changes made in this PR.

Reply:
Hi @RamyasreeChakka , I had went through and updated the check list in the PR description. PTAL!